src/runtime/string.go

```diff
@@ -283,3 +283,10 @@ func cgo_GoBytes(ptr unsafe.Pointer, length uintptr) []byte {
 	}
 	return buf
 }
+
+func cgo_CBytes(b []byte) unsafe.Pointer {
+	p := malloc(uintptr(len(b)))
+    s := unsafe.Slice(p, len(b))
```

Comment:
This results in a compiler error. I think you need this:

```suggestion
    s := unsafe.Slice((*byte)(p), len(b))
```